packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -166,34 +208,33 @@ class PopupMenuItem<T> extends PopupMenuEntry<T> {
   ///
   /// By default, the item is [enabled].
   ///
-  /// The `height` and `enabled` arguments must not be null.
+  /// The `enabled` argument must not be null.
   const PopupMenuItem({
     Key key,
     this.value,
     this.enabled = true,
-    this.height = _kMenuItemHeight,
```

Comment:
Generally we prefer having default values than using `??` because it means the type can be non-nullable in the future and it is more self-documenting (and avoids bugs like defaulting to different values in different places). Is there a strong reason to prefer null here?

Reply:
In this case it's not, I'll change it.